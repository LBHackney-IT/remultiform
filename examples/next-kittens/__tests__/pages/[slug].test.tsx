import React from "react";
import { create } from "react-test-renderer";

import SlugPage from "../../pages/[slug]";

it("renders correctly with all props", () => {
  const component = create(<SlugPage slug="big-kitten" />);

  expect(component).toMatchSnapshot();
});
