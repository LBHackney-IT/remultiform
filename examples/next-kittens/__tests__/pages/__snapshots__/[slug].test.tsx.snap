// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly with all props 1`] = `
<img
  src="https://placekitten.com/900/600"
/>
`;
